### ISSUE TITLE
Support for multiple attr_values

### ISSUE THREAD

**Issue by nirmal-suthar** [spdx/spec-parser#17]
The attributes can have multiple attr_values.
For Example `SubClassOf: Class1; Class2`

Formally it is `attr_name: attr_value ( ; attr_value )* `

**Comment by zvr** [comment]
We can even skip the `;` separator, if every value can be restricted to be a single (space-delimited) word.

Copying from an old file where I did something similar:

```
avline_list: avline | avline_list avline

avline: attrib value_list NEWLINE

value_list: value | value_list value

attrib: word
value: word

word: STRING

empty:
```

**Comment by zvr** [comment]
The COLON `:` processing will be different, since in my old example I did not use it...

**Comment by nirmal-suthar** [comment]
Can we ensure that `xsd: string` (the whitespace after colon is intentional) is invalid attr_value ?
This will be critical as we are treating space as delimiter in case of multiple attr values.

**Comment by nirmal-suthar** [comment]
> We can even skip the `;` separator, if every value can be restricted to be a single (space-delimited) word.
> 
> Copying from an old file where I did something similar:
> 
> ```
> avline_list: avline | avline_list avline
> 
> avline: attrib COLON value_list NEWLINE
> 
> value_list: value | value_list value
> 
> attrib: word
> value: word
> 
> word: STRING
> 
> empty:
> ```

**Comment by nirmal-suthar** [comment]
> The COLON `:` processing will be different, since in my old example I did not use it...
> 
> Maybe on the lexical level COLON is `\s*:\s`

So basically the difference between two colon in this example`type: xsd:string` is that
1) first colon which is `:\s` is the division between attr_name and attr_vals
2) second colon which is `:` is the regular colon which can be ignored or included in word.

**Comment by nirmal-suthar** [comment]
Resolved by #19
